src/utils/Validator.js

```diff
@@ -0,0 +1,39 @@
+export default class Validator {
+  static validateCarNames(carNames) {
```

Comment:
함수명과 관련된 내용인데요! 

저는 `validateCarNames `메서드명과 `Validator `클래스명에서 말하고자 하는 부분이 겹친다고 생각이 들어요!
`validateCarNames `는 자동차 이름을 검증하는 함수를 뜻합니다. 하지만 `Validator `클래스 자체가 검증한다 혹은 검증이라는 뜻을 포함하고 있으니, `carNames`로 메서드명을 사용하고 `Validator.carNames()`로 사용해도 `Validator.validateCarNames()`와 동일한 뜻으로 해석할 수 있고, 더 간결해진다고 생각합니다!

```suggestion
  static carNames(carNames) {
```

Reply:
그러네요! 앞 부분에 Validator가 있으니 validate가 포함되지 않아도 동일한 뜻으로 간결하게 표현되네요! 다음 번엔 이렇게 작성해봐야겠어요! 좋은 의견 감사합니다!